packages/palette/package.json

```diff
@@ -70,7 +70,7 @@
     "@types/d3-shape": "1.3.1",
     "@types/enzyme": "3.9.1",
     "@types/jest": "23.3.14",
-    "@types/luxon": "1.15.1",
+    "@types/luxon": "1.15",
```

Comment:
This is missing the caret. Not sure what that leads to, will this be pinned to 1.15.0?

Reply:
It looks like all of the @types don't have carets. Is this expected then?

Reply:
Ah, I think that’s a renovate thing. In any case, looks like we are installing at least `1.15.1` so it should be fine https://github.com/artsy/palette/blob/12597a2ec97fc760e4902229d9156aa444a9a7ef/yarn.lock#L3415